source/collaborate/learn-about-roles.rst

```diff
@@ -89,3 +89,32 @@ Mattermost is designed as a system-of-record, so there isn't an option to delete
     AD/LDAP user accounts can't be deactivated from Mattermost; they must be deactivated from your Active Directory.
 
 
+User ID
+-------
+Users can be specified in Mattermost by username or user ID. Usernames automatically resolve when a match is detected.
+Below are three ways a User's ID can be determined.
+
+Determining a User ID through the System Console (Admin)
```

Comment:
@TomerPacific - Thank you for your docs PR!

We're going to move this content out of the end user section and over to the administrative section of the product documentation because system admins are far more likely to need to determine a user's ID than an end user would.

Please move this content, including all 3 ways to determine a user's ID you've drafted, to the ``/configure/user-management-configuration-settings.rst, under the ``Users`` section. Please position it above the deactivate content as a new subsection.

As you move this content to the admin page, also use the [Deactivate user accounts](https://docs.mattermost.com/configure/user-management-configuration-settings.html#deactivate-user-accounts) content as a guide for this new content on determining User ID.